### PR TITLE
WIP: HMIS 105 Section 4 HTS Report

### PR DESCRIPTION
**Pull request by solemabrothers**
https://app.asana.com/0/1133167403049787/1134427588044389

### CODE REVIEW

File: omod/src/main/resources/apps/ugandaemr_reports_app.json [hunk 1]
```diff
@@ -352,6 +352,14 @@
         "url": "reportingui/runReport.page?reportDefinition=fe02c34a-5c67-453b-93e9-4946e7bd8c59",
         "order": 35
       },
+      {
+        "id": "HMIS 105  4",
```
Comment (ssmusoke) [inline omod/src/main/resources/apps/ugandaemr_reports_app.json]: @solemabrothers add Section to be consistent, and also move it to the 2019 monthly reports (See updated reports.json)

Comment (solemabrothers) [reply omod/src/main/resources/apps/ugandaemr_reports_app.json]: This is done

File: omod/src/main/resources/apps/ugandaemr_reports_app.json [hunk 2]
```diff
@@ -352,6 +352,14 @@
...
+        "extensionPointId": "org.openmrs.module.ugandaemr.reports.monthly",
+        "type": "link",
+        "label": "HMIS 105 SECTION 4:  HCT",
```
Comment (ssmusoke) [inline omod/src/main/resources/apps/ugandaemr_reports_app.json]: @solemabrothers change label to "HMIS 105 Section 4: HTS" which is the title of the report pages

Comment (solemabrothers) [reply omod/src/main/resources/apps/ugandaemr_reports_app.json]: OK

**Comment by ssmusoke** [comment]
@solemabrothers Please resolve the conflict here

**Comment by ssmusoke** [comment]
@solemabrothers The MER_TB_STAT.xls file should not be included in this PR

**Pull request closed without merging**